graylog2-web-interface/src/components/configurations/UrlWhiteListForm.jsx

```diff
@@ -0,0 +1,213 @@
+  };
+  const _getSummary = () => {
+    return (config.entries.map((url, idx) => {
+      return (
+      // eslint-disable-next-line react/no-array-index-key
+        <tr key={url.id}>
+          <td>{idx + 1}</td>
+          <td>
+            <Input type="text"
+                   id={`title-input${idx}`}
+                   ref={(elem) => { inputs[`title${idx}`] = elem; }}
+                   help={validationState.errors[idx] && validationState.errors[idx].title && !validationState.errors[idx].title.valid ? 'required' : null}
+                   name="title"
+                   bsStyle={validationState.errors[idx] && validationState.errors[idx].title && !validationState.errors[idx].title.valid ? 'error' : null}
+                   onChange={event => _onInputChange(event, idx, url.type)}
+                   defaultValue={url.title}
+                   required />
+          </td>
+          <td>
+            <Input type="text"
+                   id={`value-input${idx}`}
+                   ref={(elem) => { inputs[`value${idx}`] = elem; }}
+                   help={validationState.errors[idx] && validationState.errors[idx].value && !validationState.errors[idx].value.valid ? 'Not valid' : null}
+                   name="value"
+                   bsStyle={validationState.errors[idx] && validationState.errors[idx].value && !validationState.errors[idx].value.valid ? 'error' : null}
+                   onChange={event => _onInputChange(event, idx, url.type)}
+                   defaultValue={url.value}
+                   required />
+          </td>
+          <td>
+            <Input id={`url-input-type-${idx}`}
+                   required
+                   autoFocus>
+              <Select placeholder="Select Cache Type"
+                      clearable={false}
+                      options={options}
+                      matchProp="label"
+                      onChange={option => _onUpdateType(idx, option)}
+                      value={url.type} />
+            </Input>
+          </td>
+          <td>
+            <span className="">
+              <Button onClick={event => _onRemove(event, idx)}>
+                <Icon name="fa-trash" style={{ cursor: 'pointer' }} />
+              </Button>
+            </span>
+          </td>
+        </tr>
+      );
+    }));
+  };
+
+  useEffect(() => {
+    const valid = _isFormValid();
+    onUpdate(config, valid);
+  }, [config]);
+
+  return (
+    <>
+      <Input type="checkbox"
+             id="whitelist-disabled"
+             label="Disabled"
+             checked={config.disabled}
+             onChange={() => setConfig({ ...config, disabled: !config.disabled })}
+             help="Disable this white list." />
+      <Button bsSize="sm" onClick={event => _onAdd(event)}>Add Url</Button>
+      <Table striped bordered className="top-margin">
+        <thead>
+          <tr>
+            <th>#</th>
+            <th>Title</th>
+            <th>Url</th>
```

Comment:
Another nitpick: In some places we write this in uppercase as in `URL`, but other times only the first letter as in `Url`. Since it's an acronym I think we should always write it in uppercase.

Reply:
fixed

Reply:
Fixed